src/mono/netcore/sample/Android/Program.csproj

```diff
@@ -7,8 +7,6 @@
     <MicrosoftNetCoreAppRuntimePackDir>$(ArtifactsBinDir)microsoft.netcore.app.runtime.android-$(TargetArchitecture)\$(Configuration)\runtimes\android-$(TargetArchitecture)</MicrosoftNetCoreAppRuntimePackDir>
     <AndroidAppBuilderDir>$(ArtifactsBinDir)AndroidAppBuilder\$(Configuration)\$(NetCoreAppCurrent)</AndroidAppBuilderDir>
     <EnableTargetingPackDownload>false</EnableTargetingPackDownload>
-    <PublishTrimmed>true</PublishTrimmed>
```

Comment:
Should changes to the sample project be part of this PR?

Reply:
That is another PR, which was mentioned in my very first comment of this PR. This change here is a work around I took to make AndroidAppBuilder free from `publish`.